### PR TITLE
Create ReleaseOwner.md file

### PR DESCRIPTION
**Pull request by abhinavGupta16**
Signed-off-by: Abhinav Gupta <abhng@amazon.com>

### Description
This file lists down the duties to be performed by the release manager.
 
### Issues Resolved
https://github.com/opensearch-project/opensearch-build/issues/1257
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by abhinavGupta16** [comment]
This is a draft PR with initial steps and their descriptions. I will keep adding to this. 

Please feel free to commit any changes in document. Inputs are welcome.

**Comment by dblock** [comment]
This seems to mostly duplicate the content in https://github.com/opensearch-project/opensearch-build/blob/main/.github/ISSUE_TEMPLATE/release_template.md. I would extend that template as needed. There's a `<details><summary>How to use this issue</summary>` in there and you could do something similar for tasks that require more explanation. WDYT?

**Comment by abhinavGupta16** [comment]
> This seems to mostly duplicate the content in https://github.com/opensearch-project/opensearch-build/blob/main/.github/ISSUE_TEMPLATE/release_template.md. I would extend that template as needed and not create a `ReleaseOwner.md`. There's a `<details><summary>How to use this issue</summary>` in there and you could do something similar for tasks that require more explanation. WDYT?
> 
> Another option is that this content goes into `RELEASING.md`, but releasing is basically the content in README.
> 
> Finally, a third option is `RELEASE_OWNERS.md` which is similar to `OWNERS.md` that we have now.

We could add it as a part of the template, but it would be difficult to copy paste from it to create an issue in future. How would that work?

Completely unrelated, we'll need to update `release_template.md` as well.

File: ReleaseOwner.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  - [Run Integration Tests](#run-integration-tests)
+  - [Opensearch maven release](#opensearch-maven-release)
+- [Release Day](#release-day)
+- [Post Release Tasks](#post-release-tasks)
+  
+## Responsibilities
+Reponsibilities of a release manager includes but not limited to - 
+- ensuring completion of all tasks on the Release Issue
+- Keep track of any unresolved bugs and features that may affect the release timeline
+- complete the [Release Day](#release-day) Tasks
```
Comment (dblock) [inline ReleaseOwner.md:19]: The lower/upper casing is all over the place here

File: ReleaseOwner.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  - [opensearch-observability](https://github.com/opensearch-project/observability)
+  - [asynchronous-search](https://github.com/opensearch-project/asynchronous-search)
+  - [cross-cluster-replication](https://github.com/opensearch-project/cross-cluster-replication)
+  - [performance-analyzer](https://github.com/opensearch-project/performance-analyzer)
+  - [performance-analyzer-rca](https://github.com/opensearch-project/performance-analyzer-rca)
+  - [security](https://github.com/opensearch-project/security)
+  - [opensearch](https://github.com/opensearch-project/OpenSearch)
+  - [common-utils](https://github.com/opensearch-project/common-utils)
+  - [opensearch-java](https://github.com/opensearch-project/opensearch-java)
+  - [opensearch-build](https://github.com/opensearch-project/opensearch-build)
```
Comment (dblock) [inline ReleaseOwner.md:60]: This list will change. 

The content duplicates the release ticket template.

**Review by dblock — CHANGES_REQUESTED** [review]
The majority of this content is duplicate of https://github.com/opensearch-project/opensearch-build/blob/main/.github/ISSUE_TEMPLATE/release_template.md and is bound to get out of date. 

What's the reason not to modify that template with the missing content?

**Comment by abhinavGupta16** [comment]
> The majority of this content is duplicate of https://github.com/opensearch-project/opensearch-build/blob/main/.github/ISSUE_TEMPLATE/release_template.md and is bound to get out of date.
> 
> What's the reason not to modify that template with the missing content? Maybe a new section under the instructions div?

Upon further looking into the issue - The instructions are specific to the release owner only who performs these tasks. Adding all the details here explaining each task may make the issue very large with a lot details that maybe irrelevant to all the stakeholders. Alternatively, do you think we can add the this doc and link these instructions to the Issue_template?

**Comment by dblock** [comment]
https://github.com/opensearch-project/opensearch-build/pull/1612 has replaced most of this, no? @peternied @abhinavGupta16

**Comment by peternied** [comment]
> #1612 has replaced most of this, no? @peternied @abhinavGupta16

I agree.  There are items from https://github.com/opensearch-project/opensearch-build/issues/1258 that are in this PR such as the release candidate selection process that I would recommend get spun off into a new PR on the issue template

**Comment by bbarani** [comment]
> > #1612 has replaced most of this, no? @peternied @abhinavGupta16
> 
> I agree. There are items from #1258 that are in this PR such as the release candidate selection process that I would recommend get spun off into a new PR on the issue template

We are looking in to creating a new PR directly on the issue template with the suggested changes.

**Pull request closed without merging**